lib/tree_sitter.rb

```diff
@@ -1,9 +1,6 @@
 # frozen_string_literal: true
 
 # TreeSitter is a Ruby interface to the tree-sitter parsing library.
-module TreeSitter
-end
```

Comment:
Typo in the commit message.
The comment is now orphan.  You should move it below.